### DISCUSSION TITLE
v2.10.1

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1460] [Announcements]
## What's Changed
* Pin the new commit to `mason-nvim-dap` which will automatically use Python virtual environments if they are activated before opening AstroNvim

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v2.10.0...v2.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v2.10.1'>v2.10.1</a>.</em>

**Comment by tienbuigia** [comment]
There is a weird problem that if I clone astronvim and my config (.config/astronvim) then open nvim for the first time, it's going to make packer mad. It shows a pre.load error about my pywal plugin. 

I tried packersync / rm all cache or local or even nvim itself.

I fixed my problem by move my config to another place, run PackerSync then move the config back.
